app/enterprise/2.3.x/admin-api/licenses/reference.md

```diff
@@ -5,22 +5,22 @@ book: licenses
 licenses_attribute_id: |
     Attributes | Description
     ---:| ---
-    `id` | The **License's** unique ID.
+    `id` | The **license's** unique ID.
 
 licenses_body: |
     Attribute | Description
     ---:| ---
-    `payload` | The **{{site.ee_product_name}} License** in JSON format.
+    `payload` | The **Kong Gateway license** in JSON format.
 ---
 
 ## Introduction
 
-The {{site.ee_product_name}} Licenses feature is configurable through the
-[Admin API]. This feature allows the configuration a license in your
-{{site.ee_product_name}} cluster; traditional and hybrid mode deployments. In
-hybrid mode deployments, licenses configured via the `/licenses` endpoint will
-be sent to all dataplanes in the cluster and the most recent `updated_at`
-license will be used.
+The {{site.ee_base_gateway}} Licenses feature is configurable through the
```

Comment:
I just looked at the rendered documentation and `{{site.ee_base_gateway}}` is blank; is this something that coming in a different branch of the docs?

Reply:
Ack - no, that's a typo, should just be {{site.base_gateway}}